extensions/amp-story/1.0/amp-story-grid-layer.js

```diff
@@ -108,6 +108,43 @@ export class AmpStoryGridLayer extends AmpStoryBaseLayer {
     return this.prerenderAllowed_;
   }
 
+  // TODO(26866): switch back to onMeasuredChange and remove the layoutBox
+  // equality checks.
+  /** @override */
+  onLayoutMeasure() {
+    super.onLayoutMeasure();
+
+    const format = this.element.getAttribute('format');
```

Comment:
nit: I wonder if we should an attribute that more strongly indicates that this is not just the sizing of the container, but also changing how its children are sized and positioned?  Like `maintain-proportion="9:16"`?

Open to suggestions

Reply:
Maybe just `aspect-ratio="9:16"`?

Reply:
Done